backend/007-meal-plan-entry.sql

```diff
@@ -4,7 +4,7 @@ create table if not exists app.meal_plan_entry (
     id bigserial primary key,
     category app.category_t not null,
     days int not null,
-    meal_plan_id bigint REFERENCES app.meal_plan(id) NOT NULL,
+    meal_plan_id bigint REFERENCES app.meal_plan(id) ON DELETE CASCADE NOT NULL ,
```

Comment:
Delete mutation didn't work because the meal plan entry was a foreign key to the meal plan and didn't allow the meal plan record to get deleted. So added on delete cascade to meal plan entry table to delete meal plan entry when meal plan is deleted.